File: src/main/java/gg/grounds/repo/MaintenanceConfigRepository.java
package gg.grounds.repo;

import gg.grounds.domain.model.MaintenanceConfigEntity;
import io.quarkus.hibernate.reactive.panache.PanacheRepositoryBase;
import jakarta.enterprise.context.ApplicationScoped;

@ApplicationScoped
public class MaintenanceConfigRepository implements PanacheRepositoryBase<MaintenanceConfigEntity, Short> {}

File: src/main/kotlin/gg/grounds/repo/MaintenanceConfigRepository.kt
package gg.grounds.repo

import gg.grounds.domain.model.MaintenanceConfigEntity
import io.quarkus.hibernate.reactive.panache.PanacheRepositoryBase
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class MaintenanceConfigRepository : PanacheRepositoryBase<MaintenanceConfigEntity?, Short?>
